src/ol/source/tileimage.js

```diff
@@ -241,7 +241,7 @@ ol.source.TileImage.prototype.getTile = function(z, x, y, pixelRatio, projection
       !this.getProjection() ||
       !projection ||
       ol.proj.equivalent(this.getProjection(), projection)) {
-    return this.getTileInternal(z, x, y, pixelRatio, /** @type {!ol.proj.Projection} */ (projection));
+    return this.getTileInternal(z, x, y, pixelRatio, /** @type {!ol.proj.Projection} */ (this.getProjection() || projection));
```

Comment:
This function now has four `this.getProjection()` calls.  Could you please move the `sourceProjection` declaration [from below](https://github.com/pjeweb/openlayers-1/blob/b9313e76f4fed26b2fbfc953e6c3e3de6976616d/src/ol/source/tileimage.js#L257) to the top of the function and use it in the other three places?

Reply:
I created #7438 to address this.